src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier color = new ColorSupplier();
+
+    public String getRandomBall() {
+
+        Random random = new Random();
+        int number = random.nextInt(100);
+        StringBuilder ball = new StringBuilder();
+        ball.append(color.getRandomColor()).append(" ").append(number);
+        return ball.toString();

```

Comment:
you should not return string value, return an instance of Ball class
Set a color and a random number to that ball